src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -44,55 +44,56 @@ const CreateConceptForm = props => (
               onChange={props.handleChange}
             >
               <option value="N/A" />
-              {classes.map(option => <option key={option}>{option}</option>)}
+              {classes.map(option => (
+                <option key={option}>{option}</option>
```

Comment:
What happened here?

Reply:
@kmunene This is eslint fix.

Reply:
Can you please revert this formatting change as it looks like it is not part of this ticket.

Reply:
@dondrzzy I have reverted the formatting changes